owm2ffmap/requirements.txt

```diff
@@ -1,3 +1,6 @@
-tornado
```

Comment:
Can you put this in a separate commit, as it don't seem to relate to the "uplink-issue" fixed